specs/Deployment/DeploymentAPI.md

```diff
@@ -155,6 +203,16 @@ namespace Microsoft.Windows.ApplicationModel.WindowsAppRuntime
         HRESULT ExtendedError{ get; };
     };
 
+    // TODO: https://task.ms/38272182 - add APIcontract once WinAppSDK's rules for them are defined [contract(name,version)]
```

Comment:
This links to an internal ADO instance.  Are we completing and removing this before completing the PR?

Reply:
RECOMMEND: Checkin with the TODO. Update this .md spec when 38272182 is implemented